File: Neato-SDK/neato-sdk-android/src/main/java/com/neatorobotics/sdk/android/robotservices/housecleaning/HouseCleaningMinimal2Service.java
package com.neatorobotics.sdk.android.robotservices.housecleaning;

/**
 * Neato
 * Created by Marco on 04/01/2017.
 * Copyright © 2016 Neato Robotics. All rights reserved.
 */

public class HouseCleaningMinimal2Service extends HouseCleaningService {

    private static final String TAG = "HouseCleaningMinimal2";

    @Override
    public boolean isEcoModeSupported() {
        return false;
    }

    @Override
    public boolean isTurboModeSupported() {
        return false;
    }

    @Override
    public boolean isExtraCareModeSupported() {
        return true;
    }

    @Override
    public boolean isCleaningAreaSupported() {
        return false;
    }

    @Override
    public boolean isCleaningFrequencySupported() {
        return false;
    }

}

File: Neato-SDK/neato-sdk-android/src/main/java/com/neatorobotics/sdk/android/robotservices/housecleaning/HouseCleaningMinimal2Service.kt
/*
 * Copyright (c) 2019.
 * Neato Robotics Inc.
 */

package com.neatorobotics.sdk.android.robotservices.housecleaning

class HouseCleaningMinimal2Service : HouseCleaningService() {

    override val isMultipleZonesCleaningSupported: Boolean
        get() = false

    override val isEcoModeSupported: Boolean
        get() = false

    override val isTurboModeSupported: Boolean
        get() = false

    override val isExtraCareModeSupported: Boolean
        get() = true

    override val isCleaningAreaSupported: Boolean
        get() = false

    override val isCleaningFrequencySupported: Boolean
        get() = false

    companion object {

        private const val TAG = "HouseCleaningMinimal2"
    }

}
